sql-statements/sql-statement-create-placement-policy.md

```diff
@@ -23,11 +23,11 @@ PolicyName ::=
     Identifier
 
 PlacementOptionList ::=
-    DirectPlacementOption
```

Comment:
Can we do this:

```
PlacementOptionList ::=
     PlacementSugarSyntaxOptionList
   | PlacementNoSugarSyntaxOptionList

PlacementSugarSyntaxOptionList ::= "PRIMARY_REGION" EqOpt stringLit "REGIONS" EqOpt stringLit ...
```

User can easily learn how to write a right sql

Reply:
Sure